magento/magento2
# fixed condition when can show password input. Fixed logic for pulling…

… email value from persistent storage





### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#26903 : 
Guest visitor e-mail in checkout kicks off password field visibility

### Manual testing scenarios (*)

1. add items to cart and reach checkout , use a unregistered e-mail in e-mail field and fill address data
2. go back to cart page (or to any other page as long outside from checkout page)
3. then go back to checkout page and see that it presents password fields for authenticating for e-mail that is not registered

### Questions or comments


### Contribution checklist (*)
